docs/_embed/partner_api.md

```diff
@@ -140,11 +140,15 @@ The idea is to use an intermediate page that reads the access token from the URL
 
 Available parameters to the Apps resource:
 
-| parameter     | requirement | notes                                                                                  |
-| ------------- | ----------- | -------------------------------------------------------------------------------------- |
-| **client_id** | Required    | Your application client ID.                                                            |
-| **per_page**  | Optional    | (defaults to 100, max of 100) Limit the number of apps returned.                       |
-| **page**      | Optional    | (defaults to 1) The page number. Page number 1 refers to the first page in the result. |
+| parameter                   | requirement | notes                                                                                         |
+| --------------------------- | ----------- | --------------------------------------------------------------------------------------------- |
+| **client_id**               | Required    | Your application client ID.                                                                   |
+| **category**                | Optional    | Filter the results by app category.                                                           |
+| **is_in_zap_template_with** | Optional    | Service slug that limits the results to apps that are included within the same Zap Templates. |
```

Comment:
The partner does not need to specify a service slug - it doesn't matter what they specify, so long as `is_in_zap_template_with` is in the query params. I would make the notes something like

```suggestion
| **is_in_zap_template_with** | Optional    | Fetch apps that appear in a Zap Template with your app. |
```

Reply:
Looks like this got marked as resolved without changing the notes.

Reply:
So what happened was I used git's Commit Suggestions and then later accidentally forced pushed over those changes. Should be good now.